pybamm/input/parameters/lead-acid/anodes/lead_Sulzer2019/parameters.csv

```diff
@@ -36,7 +36,7 @@ Hydrogen reference OCP vs SHE [V],0,srinivasan2003mathematical,
 Negative electrode double-layer capacity [F.m-2],0.2,,
 ,,,
 # Density,,,
-Negative electrode density [kg.m-3],113400,CRC Handbook of Chemistry and Physics,
+Negative electrode density [kg.m-3],11300,CRC Handbook of Chemistry and Physics,
```

Comment:
It wasn't consistent with the value for the current collectors. Given that this one was extremely large, I assumed it was the incorrect one. @tinosulzer, does it make sense?

Reply:
Yep, just looked it up, thanks